examples/checkpointing/checkpoint.py

```diff
@@ -0,0 +1,74 @@
+from pyrevolve import Checkpoint, Operator
+from devito import TimeFunction
+
+
+class CheckpointOperator(Operator):
+    """Devito's concrete implementation of the ABC pyrevolve.Operator. This class wraps
+       devito.Operator so it conforms to the pyRevolve API. pyRevolve will call apply
+       with arguments t_start and t_end. Devito calls these arguments t_s and t_e so
+       the following dict is used to perform the translations between different names.
+       :param op: The devito.Operator object that this object will wrap
```

Comment:
If this is a wrapper, why inheriting from ``Operator`` ?

Reply:
Note that it is *inheriting* from `pyrevolve.Operator` while wrapping `devito.Operator`.

Reply:
ah, right!